tests/integration/api_v2/http/status.test.js

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
It's a good idea to have it tested, however it does not belong to the APIv2.

Reply:
Moved to separate dir `tests/integration/gateway/status.test.js`